about.html

```diff
@@ -0,0 +1,196 @@
+            <li class="nav-item">
+              <a class="item-menu-mobile"  href="./index.html">Program</a>
+            </li>
+            <li class="nav-item">
+              <a class="item-menu-mobile" aria-current="page" href="./index.html">Join</a>
+            </li>
+            <li class="nav-item">
+              <a class="item-menu-mobile" href="#sponsor">Sponsor</a>
+            </li>
+            <li class="nav-item">
+              <a class="item-menu-mobile" href="#news">News</a>
+            </li>
+            <li class="nav-item">
+              <a class="item-menu-mobile" href="#news">CC Campaing</a>
+            </li>
+          </ul>
+        </nav>
+      </div>  <!-- menu movil-->    
+    </div>
+  </header>
+
+  <main>
+    <div class="container hero p-4">
+      <div class="about-hero-title text-center">
+        <h4 class="title-hello">Hello! Music lovers</h4>
+        <h1 class="title-mmw">MIAMI MUSIC<br>
+          WEEKEND 2023<br>
+        </h1>
+        <h6 class="title-slogan">Where Music, Tech + Biz Converge</h6>
+        <p class="p-3 title-descrip">
+          We promise to deliver a world class experience in the heart of Miami Beach once again. MMW is an annual entertainment event dedicated to the expression and growth of the Rock and pop music, media and entertainment industries. 
+          Combining multifaceted information-intensive conferences; a trade exposition; awards shows and the nation’s largest new music festival which spans 3 nights of in-person and virtual performances (June 22-25 2025) with hundreds
+          of bands.
+          The Conference will be held June 22-23, 2023 in a new location this year: Greater Miami Convention & Visitors Bureau.
+        </p>
+        <h5 class="title-date">Please contac us</h5>
+        <p class="title-place">1 Hillboroug Square, Miami Beach<br>Telephone: 01-434-589-7600.</p>
+      </div>
+    </div>
+
+    <div class="container bg-light p-4">
+      <h3 class="title-page text-dark text-center pt-3">
+        Gallery
+      </h3>    
+      <div class="short-line-center pb-5"></div>
+
+      <div class="d-md-flex">        
+        <div class="card col-12 col-md-6 text-white border-0 py-1 mx-md-1">
+          <img src="./images/mmw2019.jpg" class="img-fluid img-gallery" alt="Miami Music Weekend 2019 photo.">
+          <div class="card-img-overlay text-center">
+            <h5 class="gal-card-title">MMW 2019</h5>
+            <p class="gal-card-text">relive some of the magical moments</p>
+          </div>
+        </div>
+        <div class="card col-12 col-md-6 text-white border-0 py-1 mx-md-1">
+          <img src="./images/mmw2020.jpg" class="img-fluid img-gallery" alt="Miami Music Weekend 2020 photo.">
+          <div class="card-img-overlay text-center">
+            <h5 class="gal-card-title">MMW 2019</h5>
+            <p class="gal-card-text">the iconic festival that everyone remembers</p>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ]  Please kindly use semantic tag, put this part into a `<section>` tag for HTML best practice 😁
- [ ] Also as you see in the original design the image has an opacity and before the image we have a line of text, kindly fix that 🙃

Your design

<img width="871" alt="Screenshot 2023-04-27 at 00 59 07" src="https://user-images.githubusercontent.com/74501523/234712524-e23a64aa-8dcf-446a-89af-d903f0510da4.png">

Original design

<img width="871" alt="Screenshot 2023-04-27 at 00 59 37" src="https://user-images.githubusercontent.com/74501523/234712584-61581c83-b111-4da1-aea9-9f08c542a98e.png">

- [ ] And I noticed that you literally miss one section that comes before the `Galery section`, this section should contain a middle section with a central column that includes a section header, a divider, a paragraph of text, and a large logo. Kindly add it 🙏 

Original design

<img width="871" alt="Screenshot 2023-04-27 at 01 03 06" src="https://user-images.githubusercontent.com/74501523/234713071-801e06d5-cb9c-48fa-9054-a1a6ec565f03.png">

Reply:
thank you. Changes done.